Documentation/Contributing.md

```diff
@@ -128,40 +128,41 @@ It is used on bare systems to bootstrap the Swift toolchain (including SwiftPM),
 the bootstrap script requires having [CMake](https://cmake.org/) and [Ninja](https://ninja-build.org/) installed.
 Please refer to the [Swift project repo](https://github.com/apple/swift/blob/master/README.md#macos) for installation instructions.
 
-1. Clone [llbuild](https://github.com/apple/swift-llbuild) beside the SwiftPM directory.
-
-```bash
-$> git clone https://github.com/apple/swift-llbuild llbuild
-```
-
-Note: Make sure the directory for llbuild is called "llbuild" and not
-    "swift-llbuild".
-
-2. Clone [Yams](https://github.com/jpsim/yams) beside the SwiftPM directory.
-
-```bash
-$> git clone https://github.com/jpsim/yams
-```
-
-3. Clone [swift-driver](https://github.com/apple/swift-driver) beside the SwiftPM directory.
-
-```bash
-$> git clone https://github.com/apple/swift-driver
-```
-
-4. Clone [swift-argument-parser](https://github.com/apple/swift-argument-parser) beside the SwiftPM directory and check out tag with the [latest version](https://github.com/apple/swift-argument-parser/tags).
-
-For example, if the latest tag is 0.3.1:
-```bash
-$> git clone https://github.com/apple/swift-argument-parser --branch 0.3.1
-```
-
-5. Clone [swift-crypto](https://github.com/apple/swift-crypto) beside the SwiftPM directory and check out tag with the [latest version](https://github.com/apple/swift-crypto/tags).
-
-For example, if the latest tag is 1.1.3:
-```bash
-$> git clone https://github.com/apple/swift-crypto --branch 1.1.3
-```
+1. Clone [llbuild], [swift-driver], and [Yams] beside the SwiftPM directory.
+   
+   ```sh
+   $> git clone https://github.com/apple/swift-llbuild llbuild
+   $> git clone https://github.com/apple/swift-driver
+   $> git clone https://github.com/jpsim/yams
+   
+   ```
+   
+   > **Note**: Make sure the directory for llbuild is called "llbuild" and not "swift-llbuild".
+   
+   [llbuild]: https://github.com/apple/swift-llbuild
+   [swift-driver]: https://github.com/apple/swift-driver
+   [Yams]: https://github.com/jpsim/yams
```

Comment:
I took the liberty of consolidating these 3 `git clone` steps while adding instruction for cloning swift-collections to the documentation.

Would it be a good idea to change the prompt from "$>" to ";" in a follow-up PR? Using ";" will allow the commands to be easily copy-pasted. However, it's not commonly used, and all prompts in this document will have to be changed for consistency.

Reply:
I think it's clearer if they are kept separate, even if it does mean copying and pasting three different cases.  If combined, I don't think `;` should necessary be at the beginning.  Something like this might make sense:
```
git clone https://github.com/apple/swift-llbuild llbuild ; \
git clone https://github.com/apple/swift-driver ; \
git clone https://github.com/jpsim/yams
```

Reply:
In general I've found it difficult to come up with a character that indicates a prompt without having something think they should include it in the command.  I've seen `❯` used, which is more clearly not part of command, but whatever is done here should be consistent with the rest of the document.

I think that it's probably wise to leave things as they are for this PR and if doing reformatting, to open a separate PR for that.

Reply:
Ok. I'll change them back to 3 separate steps.

Reply:
I changed the instructions back to separate steps.